java-frontend/src/main/java/org/sonar/java/se/checks/UnclosedResourcesCheck.java

```diff
@@ -404,24 +419,20 @@ public void visitMethodInvocation(MethodInvocationTree syntaxNode) {
       }
       if (syntaxNode.methodSelect().is(Tree.Kind.MEMBER_SELECT) && needsClosing(syntaxNode.symbolType())) {
         final ExpressionTree targetExpression = ((MemberSelectExpressionTree) syntaxNode.methodSelect()).expression();
+        boolean jdbcResourceCreation = isJdbcResourceCreation(syntaxNode);
```

Comment:
No need for a dedicated method anymore, I would inline it as it's the only remaining usage. I also think that calling `JDBC_RESOURCE_CREATIONS.anyMatch(syntaxNode)` is explicit enough with the assignment.